### PR TITLE
Fix/Add Artifacts with addComponent

### PR DESCRIPTION
**Pull request by bjoernmu**
Signed-off-by: Björn Müller <bjoern.mueller@kanu-baerchen.de>

-Fix wrong namespace when adding an artifact template to a nodetype implementation
-addComponent component for adding an artifact template

- [x] Ensure that you followed http://eclipse.github.io/winery/dev/ToolChain#github---prepare-pull-request. Especially, we require **a single commit**
- [x] Ensure that the commit message is [a good commit message](https://github.com/joelparkerhenderson/git_commit_message)
- [x] Ensure to use auto format in **all** files
- [x] Ensure that you appear in `NOTICE` at Copyright Holders
- [ ] Tests created for changes
- [ ] Documentation updated (if needed)
- [ ] Screenshots added (for UI changes)

### CODE REVIEW

**Comment by genie-winery** [comment]
Can one of the admins verify this patch?

File: org.eclipse.winery.frontends/app/tosca-management/src/app/wineryAddComponentDataModule/addComponentData.module.ts [hunk 1]
```diff
@@ -0,0 +1,60 @@
+/*******************************************************************************
+ * Copyright (c) 2020 Contributors to the Eclipse Foundation
```
Comment (lharzenetter) [inline org.eclipse.winery.frontends/app/tosca-management/src/app/wineryAddComponentDataModule/addComponentData.module.ts:2]: Please remove this license header and update the other one.

File: org.eclipse.winery.frontends/app/tosca-management/src/app/wineryAddComponentDataModule/addComponentData.component.html [hunk 2]
```diff
@@ -0,0 +1,139 @@
+<!--~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+  ~ Copyright (c) 2020 Contributors to the Eclipse Foundation
```
Comment (lharzenetter) [inline org.eclipse.winery.frontends/app/tosca-management/src/app/wineryAddComponentDataModule/addComponentData.component.html:2]: As this code is copyßpaste, use 2017-2020

File: org.eclipse.winery.frontends/app/tosca-management/src/app/wineryAddComponentDataModule/addComponentData.component.ts [hunk 3]
```diff
@@ -0,0 +1,176 @@
+/*******************************************************************************
+ * Copyright (c) 2020 Contributors to the Eclipse Foundation
```
Comment (lharzenetter) [inline org.eclipse.winery.frontends/app/tosca-management/src/app/wineryAddComponentDataModule/addComponentData.component.ts:2]: As this code is copyßpaste, use 2017-2020

**Pull request merged**